daemon/container_operations.go

```diff
@@ -27,6 +27,7 @@ var (
 	// ErrRootFSReadOnly is returned when a container
 	// rootfs is marked readonly.
 	ErrRootFSReadOnly = errors.New("container rootfs is marked read-only")
+	getPortMapInfo    = container.GetSandboxPortMapInfo
```

Comment:
Because if I had to invoke it specifying the package as `container.GetSandboxPortMapInfo()`, it would conflict with the variable `container` used in the functions. The other route would be to rename the pkg import. But both the solutions would consist in a lot of unneeded lines of code change.